### ISSUE TITLE
Error: Unrecognized chain name or RPC url: rinkeby

### ISSUE THREAD

**Issue by talhaa99** [thirdweb-dev/js#355]
I got this error after upgrading to @thirdweb-dev/sdk to 3.4.2 and @thirdweb-dev/react to 2.9.7

`Server Error
Error: Unrecognized chain name or RPC url: rinkeby

This error happened while generating the page. Any console logs will be displayed in the terminal window.
Call Stack
Object.getProviderForNetwork
file:///Users/apple/Documents/GitHub/safrootics-app-react/node_modules/@thirdweb-dev/react/node_modules/@thirdweb-dev/sdk/dist/thirdweb-dev-sdk.cjs.dev.js (1195:15)
<unknown>
file:///Users/apple/Documents/GitHub/safrootics-app-react/node_modules/@thirdweb-dev/react/dist/thirdweb-dev-react.cjs.dev.js (745:49)
Array.reduce
<anonymous>
<unknown>
file:///Users/apple/Documents/GitHub/safrootics-app-react/node_modules/@thirdweb-dev/react/dist/thirdweb-dev-react.cjs.dev.js (744:30)
Object.useMemo
file:///Users/apple/Documents/GitHub/safrootics-app-react/node_modules/react-dom/cjs/react-dom-server.node.development.js (1582:19)
Object.useMemo
file:///Users/apple/Documents/GitHub/safrootics-app-react/node_modules/react/cjs/react.development.js (1532:21)
ThirdwebProvider
file:///Users/apple/Documents/GitHub/safrootics-app-react/node_modules/@thirdweb-dev/react/dist/thirdweb-dev-react.cjs.dev.js (743:28)
processChild
file:///Users/apple/Documents/GitHub/safrootics-app-react/node_modules/react-dom/cjs/react-dom-server.node.development.js (3353:14)
resolve
file:///Users/apple/Documents/GitHub/safrootics-app-react/node_modules/react-dom/cjs/react-dom-server.node.development.js (3270:5)
ReactDOMServerRenderer.render
file:///Users/apple/Documents/GitHub/safrootics-app-react/node_modules/react-dom/cjs/react-dom-server.node.development.js (3753:22)`

**Comment by nachoiacovino** [comment]
hello @talhaa99 , can you also update react to 3.4.2?

if you're using Rinkeby contracts, keep in mind the chain has been deprecated and it won't work anymore.

**Comment by ass77** [comment]
> hello @talhaa99 , can you also update react to 3.4.2?
> 
> if you're using Rinkeby contracts, keep in mind the chain has been deprecated and it won't work anymore.

I encountered the same issue, it broke my app in both the development and production phases.

I was using react and SDK version `2.9` and using both Mumbai and Polygon contracts for different deployment phases. 

Then I upgraded the `@thirdweb-dev/react` and @thirdweb-dev/sdk` both to version `3.4.4`  with [migrations](https://portal.thirdweb.com/react/migration-guide) and now it works fine.
